packages/patternfly-4/react-core/src/components/Avatar/Avatar.js

```diff
@@ -4,15 +4,16 @@ import { css } from '@patternfly/react-styles';
 import PropTypes from 'prop-types';
 
 const propTypes = {
+  /** Additional classes added to the Avatar. */
```

Comment:
Change Avatar to Brand. Same comment for the other props.

Reply:
@tlabaj this is for the avatar component. I had the prop comments in the wrong spot in my last PR so I corrected it in this one.

Reply:
Got it!